mondrian/src/it/java/mondrian/test/DrillThroughTest.java

```diff
@@ -930,7 +930,7 @@ public void testDrillThroughDupKeysAndMeasure() throws Exception {
            result.getCell(new int[]{0, 0}).getDrillThroughSQL(true);
 
        testContext.assertSqlEquals(
-           "select store.frozen_sqft as Frozen sqft, store.grocery_sqft as Grocery sqft, store.meat_sqft as Meat sqft, store.store_sqft as Store sqft, store.store_sqft as Store sqft_0 from foodmart.store as store where store.frozen_sqft = 2452 order by store.frozen_sqft ASC, store.grocery_sqft ASC, store.meat_sqft ASC, store.store_sqft ASC",
+           "select store.frozen_sqft as Frozen sqft, store.grocery_sqft as Grocery sqft, store.meat_sqft as Meat sqft, store.store_sqft as Store sqft, store.store_sqft as Store sqft_0 from foodmart.store as store where store.frozen_sqft = 2452 order by Frozen sqft ASC, Grocery sqft ASC, Meat sqft ASC, Store sqft ASC",
```

Comment:
That doesn't look right. Why did it go from a column name to a column label? This SQL wouldn't work.

Reply:
I was concerned about the syntax myself, but here's the reason for the aliases: MySQL went from version 5.6 to 8.0;
https://github.com/pentaho/mondrian/blob/master/mondrian/src/main/java/mondrian/spi/impl/MySqlDialect.java#L359

Does this mean we have a more general problem with MySQL 8?  In the original SQL the aliases have spaces in them which looks wrong to me as well; couldn't find anything in the MySQL docs that say for sure if that's legal or not.